### PR TITLE
fix: update logic to conditionally show Get Started and Billing routes

### PR DESCRIPTION
**Pull request by YounixM**

### CODE REVIEW

File: frontend/src/hooks/useLicense/constant.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export const LICENSE_PLAN_KEY = {
 	ENTERPRISE_PLAN: 'ENTERPRISE_PLAN',
-	BASIC_PLAN: 'BASIC_PLAN ',
+	BASIC_PLAN: 'BASIC_PLAN',
```
Comment (palashgdev) [inline frontend/src/hooks/useLicense/constant.ts:3]: noice

File: frontend/src/container/SideNav/SideNav.tsx [hunk 2]
```diff
@@ -44,36 +51,45 @@ function SideNav(): JSX.Element {
...
-
-				if (!isOnboardingEnabled || !isCloudUser()) {
-					return item.key !== ROUTES.GET_STARTED;
-				}
-
-				return true;
-			}),
-		[featureResponse.data, isOnBasicPlan, role],
-	);
+	useEffect((): void => {
```
Comment (palashgdev) [inline frontend/src/container/SideNav/SideNav.tsx:58]: two useEffect for setMenu as eventually both would be running and overriding one over another can we make one util?

**Review by palashgdev — DISMISSED** [review]

File: frontend/src/container/SideNav/SideNav.tsx [hunk 3]
```diff
@@ -44,36 +51,45 @@ function SideNav(): JSX.Element {
...
+		if (!isFetching) {
+			let items = [...menuItems];
+
+			const isOnBasicPlan =
+				data?.payload?.licenses?.some(
+					(license) =>
+						license.isCurrent && license.planKey === LICENSE_PLAN_KEY.BASIC_PLAN,
+				) || data?.payload?.licenses === null;
+
+			if (role !== 'ADMIN' || isOnBasicPlan) {
```
Comment (Rajat-Dabade) [inline frontend/src/container/SideNav/SideNav.tsx:85]: Can we use constant `ADMIN` from the `src/types/roles.ts` file? Instead of a hardcoded string.

**Review by Rajat-Dabade — APPROVED** [review]
One small change, everything else LGTM.

**Review by Rajat-Dabade — APPROVED** [review]
LGTM 🚀

**Review by palashgdev — APPROVED** [review]
LGTM

**Pull request merged**